dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -847,6 +850,26 @@ ERROR_INCOMPATIBLE_FEATURES_SINGLETON=CWWKF1405E: The {0} and {1} singleton feat
 ERROR_INCOMPATIBLE_FEATURES_SINGLETON.explanation=The features cannot be installed at the same time.
 ERROR_INCOMPATIBLE_FEATURES_SINGLETON.useraction=Remove one of the features and try again.
 
+ERROR_FAILED_TO_UNPACK_LICENSE=CWWKF1406E: Unable to unzip the license package with the {0} Maven coordinate from the repository at the {1} file path.
+ERROR_FAILED_TO_UNPACK_LICENSE.explanation=Unable to unzip the license package to the wlp installation.
+ERROR_FAILED_TO_UNPACK_LICENSE.useraction=Ensure you have the correct permissions to read from the repository.
+
+ERROR_FAILED_TO_READ_LICENSE=CWWKF1407E: Unable to process the license file at the {0} file location.
+ERROR_FAILED_TO_READ_LICENSE.explanation=Unable to process the license file for license acceptance information.
+ERROR_FAILED_TO_READ_LICENSE.useraction=Ensure you have the correct permissions to read from the license file.
+
+ERROR_FAILED_TO_FIND_WEBSPHERE_JSON=CWWKF1408E: Unable to find the Websphere JSON file at the {0} file location.
+ERROR_FAILED_TO_FIND_WEBSPHERE_JSON.explanation=Unable to find Websphere JSON file.
+ERROR_FAILED_TO_FIND_WEBSPHERE_JSON.useraction=Ensure that you have network connection and write permissions to the configured local Maven directory.
+
+ERROR_FAILED_TO_LOCATE_AND_DOWNLOAD_JSONS=CWWKF1409E: Unable to find the following feature JSON files locally or on the configured Maven repository: {0}.
+ERROR_FAILED_TO_LOCATE_AND_DOWNLOAD_JSONS.explanation=The JSON files cannot be found locally or on the configured Maven repository.
+ERROR_FAILED_TO_LOCATE_AND_DOWNLOAD_JSONS.useraction=Ensure that you have network connection and the JSON files exist on the configured Maven repository.
+
+ERROR_INVALID_PRODUCT_EDITION_FEATURE_UTILITY=CWWKF1410E: The feature {0} cannot be installed to {1} {2} edition because it only applies to the editions {3}
```

Comment:
Please use the `the {variable} noun` format. Thus, use `the {0} feature` and possibly ` the {3} editions`.